addons/point_of_sale/static/src/app/navbar/navbar.js

```diff
@@ -112,8 +113,17 @@ export class Navbar extends Component {
             this.notification.add("Navigate to your POS Customer Display on the other computer");
         }
         if (this.pos.config.customer_display_type === "proxy") {
-            this.notification.add("Connecting to the IoT Box");
             const proxyIP = this.pos.getDisplayDeviceIP();
+            if (!proxyIP) {
+                this.dialog.add(AlertDialog, {
+                    title: _t("Connection error"),
+                    body: _t(
+                        "Make sure you specify the display device in the configuration.\n\nLook for\n1) 'Connected Devices > IoT Box > Customer Display' if you activated the enterprise 'pos_iot' module, or for\n2) 'Connected Devices > IoT Box > IoT Box IP Address' if you didn't."
```

Comment:
why would we even allow a user to configure a customer display in proxy mode, but not set it up ? Maybe we should add a constrain in the config

Reply:
Let me try that.